src/main/java/edu/kis/vh/nursery/listAndArray/IntStackList.java

```diff
@@ -0,0 +1,15 @@
+package edu.kis.vh.nursery.listAndArray;
+
+public interface IntStackList {
```

Comment:
Names of operations do not correspond to operating the stack, thus do not reflect the interface name.